usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -20,11 +20,11 @@ SELECT
   END AS display_award_id,
 
   CASE
-    WHEN AWD.update_date > TN.update_date THEN AWD.update_date
-    ELSE TN.update_date
+    WHEN UTM.award_update_date > UTM.update_date THEN UTM.award_update_date
+    ELSE UTM.update_date
   END AS update_date,
```

Comment:
Rather than a `CASE` statement here, I think this may be more efficient -- but needs testing to make sure it doesn't get worse. It is definitely more readable:

```sql
GREATEST(UTM.update_date, UTM.award_update_date) AS update_date
```

Reply:
Done

Reply:
I don't think it will noticeably change performance, one way or the other